backend/controllers/social.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+import socialModel, { ISocial } from '../models/social.model';
+
+export const getAllSocials = async (_req: Request, res: Response) => {
+  try {
+    const socials = await socialModel.find();
+    res.json(socials);
```

Comment:
why you don't check response.status? Maybe you can check length response to make sure you have any element in array?

Reply:
@Krysik try to use custom HTTP Response Code to throw custom error code when we will get empty response 

```socials.length === 0```

Reply:
Should I also check if `req.body` is empty in post endpoints ?

Reply:
No, it will be checked by schema validation

Reply:
Why We can't just return an empty array ?

Reply:
Maybe it's not suitable for our case because we always have socials and users. Let's take a look further situation. We have an endpoint `GET /socials` which return values. We want to populate options in dropdown using these values. When we will receive 2 results, its fine, we will see 2 options in dropdown. What has happened when we will receive 0 results? Dropdown will be empty and from User perspective its a bug. Example in image

<img width="826" alt="Screenshot 2021-03-26 at 23 14 42" src="https://user-images.githubusercontent.com/19636010/112698031-15a91b80-8e89-11eb-9267-6cd8c27b8eec.png">

Reply:
Ok, thanks for explanation